tools/code_format/rst_check.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import subprocess
+import sys
+
+# TODO(phlax): add rstcheck also
+
+# things we dont want to see in generated docs
+# TODO(phlax): move to .rstcheck.cfg when available
```

Comment:
You have another WiP PR here; a bit confused..

Reply:
so the point was to move it there once that PR landed

i wanted to land this because it fixes actual problems, and then add a better solution in the follow up #15786

Reply:
Can you let me know when the Python is ready for final review?

Reply:
/wait